TestCntlrApp/src/enbApp/nb.c

```diff
@@ -725,6 +725,8 @@ NbConfigReq   *cfg
    smCfgCb.s1SetupTmrVal       = cfg->s1SetupTmr;
    /* get the plmn-id */
    plmnLen = strlen((S8*)cfg->plmnId);
+   /* limit strlen as struct doesnt guarantee a null char after the str */
+   plmnLen = plmnLen > 6 ? 6 : plmnLen;
```

Comment:
The code adds a check here for PLMN string should not be greater than 6. The **plmnLen** is calculated here from **cfg->plmnId** which comes from **src/tfwApp/fw_api_int.c** and you have already made change there as part of this PR that it will always copy maximum 6 characters only.

**So do we need this check?**

Even if we need this check, instead of 6, its better to use already defined macros like "**NBT_MAX_PLMN_ID**"

Reply:
I have switched this to instead use the `strnlen` function with the `NBT_MAX_PLMN_ID` define. This check was needed as `strlen` will happily continue reading past the end of `plmnId` until it finds a null byte. `strlen` does not care what the definition of the struct is as it deals with raw underlying bytes/memory.

Reply:
Thank you for the change. **The change looks good to me.**

However, since you have modified the code for **enbName** also in file "**src/tfwApp/fw_api_int.c**". I would appreciate if you can make similar change as above for **enbName** also in line numbers 716/720 also.